pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -364,6 +364,9 @@ type PMMSpec struct {
 	ContainerSecurityContext *corev1.SecurityContext `json:"containerSecurityContext,omitempty"`
 
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// PMM cluster name. If not set Operator uses cr.Name for PMM cluster name.
+	CustomClusterName string `json:"customReplsetName,omitempty"`
```

Comment:
Trying to understand why the variable has a different name than the field we will use in yaml. Do we have a specific reason to keep them separate?

Reply:
no, you found a bug 😄